bedrock/exp/templates/exp/firefox/mobile.html

```diff
@@ -38,19 +38,14 @@
     <header class="mzp-c-hero mzp-t-firefox mzp-has-image t-hero-primary">
```

Comment:
The forms now seems to be working, but the /exp/ page has no Fluent file associated with it, so it shows string IDs instead of copy:

![image](https://user-images.githubusercontent.com/400117/93242438-b56d4a80-f77e-11ea-8590-1b9acbd2c3e0.png)